docs/siem/detections/detection-engine-intro.asciidoc

```diff
@@ -1,26 +1,59 @@
 [[detection-engine-overview]]
 [role="xpack"]
 
-= Detections (beta)
+= Detections and Alerts (beta)
 
 beta[]
 
-The SIEM Detections feature automatically searches for threats and creates 
-signals when they are detected. Signal detection rules define the conditions 
-for creating signals. Additionally, you can use the {kib}
-{kibana-ref}/alerting-getting-started.html[Alerting and Actions]
-framework to send notifications via other systems, such as email and Slack,
-when signals are generated.
+Use the Detections feature to create and manage rules, and view the alerts
+these rules create. Rules periodically search indices (such as `endgame-*` and
+`filebeat-*`) for suspicious source events, and create alerts when a rule's
+conditions are met. When an alert is created, its status is `Open`. To help
+track investigations, an alert's status can be set as `Open`, `In progress`, or
+`Closed` (see <<detection-alert-status>>).
 
-NOTE: To use {kib} Alerting for signal notifications, you need the
+[role="screenshot"]
+image::images/detections-ui.png[]
+
+In addition to creating <<rules-ui-create, your own rules>>, enable
+<<load-prebuilt-rules, Elastic prebuilt rules>> to immediately start detecting
+suspicious activity. For detailed information on all the prebuilt rule, see the
+<<prebuilt-rules>> section. Once the prebuilt rules are loaded and
+running, <<tuning-detection-signals>> and <<detections-ui-exceptions>> explain
+how to modify the rules to reduce false positives and get a better set of
+actionable alerts. You can also use exceptions and value lists when creating or
+modifying your own rules.
+
+There are two special prebuilt rules you need to know about:
+
+* *Elastic Endpoint Security*: Automatically creates an alert from all incoming
+Elastic Endpoint alerts. To receive Elastic Endpoint alerts, you must install
+the Endpoint agent on your hosts (BEN: see xref).
++
+When this rule is enabled, the following Endpoint events are displayed as
+detection alerts:
++
+** Malware Prevention Alert
+** Malware Detection Alert
++
+NOTE: When you load the prebuilt rules, this is the only rule that is enabled
+by default.
+
+* *External Alerts*: Automatically creates an alert for all incoming
+third-party system alerts (for example, Suricata alerts).
```

Comment:
Not sure if worth mentioning the explicit field, but this rule will create detection alerts for all events of `event.kind:alert` and excludes `event.module:(endgame or endpoint)` to not create duplicates of the endpoint alerts.

Reply:
Added links to both rule descriptions instead